src/main/kotlin/io/snyk/plugin/snykcode/SnykCodeBulkFileListener.kt

```diff
@@ -28,50 +29,71 @@ class SnykCodeBulkFileListener : SnykBulkFileListener() {
         CacheBuilder.newBuilder()
             .expireAfterWrite(Duration.ofMillis(1000)).build<String, Boolean>()
 
+    private val blackListedDirectories =
```

Comment:
This makes sure, we don't run scans on changes within these directories. Especially the .idea directory was very active.